src/main/java/org/example/Employee.java

```diff
@@ -0,0 +1,27 @@
+package main.java.org.example;
+
+public class Employee {
+    private String name;
+    private String department;
+
+    public Employee(String name, String department) {
+        this.name = name;
+        this.department = department;
+    }
+
+    public String getName() {
+        return name;
```

Comment:
You can improve getters for cleaner code style

Reply:
hmm.. what do you mean exactly? i have used generate -> getters and setters so it was done automatically. Is there something wrong with this code?

Reply:
Nothing is wrong. Are you using SonarLint?

Reply:
If not install that plugin and when you format code with it, you`ll see that best practice is to use this.name even in getters

Reply:
oh i get the point now your right! i did this by using generate -> setters and getters i thought it did that automatically, didnt look on it...